config/db.config.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    HOST: process.env.DATABASE_HOST || "localhost",
+    USER: process.env.DATABASE_USER || "root",
```

Comment:
Nope, this is not good. We won't run our database with root privileges because a potential SQL Injection will give the attacker the possibility to do whatever they want. As with Linux users, we can create separate users for the database and assign individual permissions. I will create a new user, but we need to discuss what that user can and should do and what not.

Reply:
It's super okay. Just make sure to set that environment variable otherwise. That is for me to test locally.